### PR TITLE
Frontend albina

### PR DESCRIPTION
**Pull request by AlbinaPavlova**

### CODE REVIEW

File: iTechArt.UI/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const routes: Routes = [
+  {path: '', component: DashboardPageComponent },
+  {path: 'med-staff', component: MedStaffPageComponent},
+  {path: 'police', component: PolicePageComponent },
+  {path: 'students', component: StudentsPageComponent},
+  {path: 'pupils', component: PupilsPageComponent },
+  {path: 'grocery', component: GroceryPageComponent},
+  {path: 'airport', component: AirportPageComponent }
+
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/app-routing.module.ts]: delete useless row

Comment (AlbinaPavlova) [reply iTechArt.UI/src/app/app-routing.module.ts]: done:)

File: iTechArt.UI/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
```
Comment (ShpilevskyAndrei) [inline iTechArt.UI/src/app/app.component.spec.ts:2]: split import's rows (internal and external)

File: iTechArt.UI/src/app/shared/components/header/header.component.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import { UNITS } from '../../constants/units';
+import { UnitEnum } from '../../enums/unit.enum';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent {
+  public units = UNITS;
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/components/header/header.component.ts]: add type

File: iTechArt.UI/src/app/shared/components/header/header.component.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  public unitsTitles = UnitEnum;
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/components/header/header.component.ts]: add type

File: iTechArt.UI/src/app/shared/constants/apis.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  //   export: 'api/pupils/export',
+  // },
+  // students: {
+  //   import: 'api/student/import',
+  //   export: 'api/student/export',
+  // },
+  // airport: {
+  //   import: 'api/airport/import',
+  //   export: 'api/airport/export',
+  // },y
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/apis.ts]: return apis. they will come in handy in the near future

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+export const UNITS: Record<UnitEnum, UnitTypeInterface> = {
+  police: {
+    title: 'Police',
+    number: Math.floor(Math.random() * 1000),
+    icon: '../../assets/police.png',
+    path: 'police',
+  },
+  students: {
+    title: 'Students',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: delete unnecessary row 'number'

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 7]
```diff
@@ -0,0 +1,41 @@
+import { UnitTypeInterface } from '../interfaces/unit-type.interface';
+import { UnitEnum } from '../enums/unit.enum';
+
+export const UNITS: Record<UnitEnum, UnitTypeInterface> = {
+  police: {
+    title: 'Police',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: delete unnecessary row 'number'

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  students: {
+    title: 'Students',
+    number: Math.floor(Math.random() * 1000),
+    icon: '../../assets/students.png',
+    path: 'students',
+  },
+  pupils: {
+    title: 'Pupils',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: same

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    icon: '../../assets/pupils.png',
+    path: 'pupils',
+  },
+  airport: {
+    title: 'Airport',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: same

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    icon: '../../assets/airport.png',
+    path: 'airport',
+  },
+  medStaff: {
+    title: 'MedStaff',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: same

File: iTechArt.UI/src/app/shared/constants/units.ts [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    icon: '../../assets/med.png',
+    path: 'med-staff',
+  },
+  grocery: {
+    title: 'Grocery',
+    number: Math.floor(Math.random() * 1000),
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/constants/units.ts]: same

File: iTechArt.UI/src/app/shared/enums/unit.enum.ts [hunk 12]
```diff
@@ -0,0 +1,8 @@
+export enum UnitEnum {
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/enums/unit.enum.ts]: rename to UnitsEnum

File: iTechArt.UI/src/app/shared/interfaces/unit-type.interface.ts [hunk 13]
```diff
@@ -0,0 +1,6 @@
+export interface UnitTypeInterface {
+  title: string,
+  number: number | string,
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/interfaces/unit-type.interface.ts]: delete rudiment

File: iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.scss [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+    font-weight: bold;
+  }
+}
+
+.dashboard-page {
+  display: flex;
+  justify-content: space-around;
+}
+
+//.airport{
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.scss]: delete comments

File: iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts [hunk 15]
```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit } from '@angular/core';
+
+import { UNITS } from '../../constants/units';
+import { HttpClient } from '@angular/common/http';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts]: upper

File: iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+  public ngOnInit(): void {
+    this.getUnitCountsInfo();
+  }
+
+  public getUnitCountsInfo(): void {
+    this.http
+      .get<any>( 'https://itechart-app.herokuapp.com/api/dashboard/total-amounts')
+      .subscribe((data) => {
+        this.unitCountsInfo = data;
+        console.log('TOTALS-AMOUNTS received: ', data);
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts]: delete console.log

File: iTechArt.UI/src/app/shared/shared.module.ts [hunk 17]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { HeaderComponent } from './components/header/header.component';
+import { DashboardPageComponent } from './pages/dashboard-page/dashboard-page.component';
+import { RouterLink } from '@angular/router';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/shared/shared.module.ts]: upper

File: iTechArt.UI/src/app/stats/modules/grocery/grocery.module.ts [hunk 18]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/stats/modules/grocery/grocery.module.ts:2]: add row between internal and external imports

File: iTechArt.UI/src/app/stats/modules/grocery/pages/grocery-page/grocery-page.component.ts [hunk 19]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-grocery-page',
+  templateUrl: './grocery-page.component.html',
+  styleUrls: ['./grocery-page.component.scss'],
+})
+export class GroceryPageComponent  {
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/stats/modules/grocery/pages/grocery-page/grocery-page.component.ts]: rows by numbers 8 + 9 + 10 => to single string

File: iTechArt.UI/src/app/stats/modules/medStaff/med-staff.module.ts [hunk 20]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/stats/modules/medStaff/med-staff.module.ts:2]: adde row between internal and external imports

File: iTechArt.UI/src/app/stats/modules/police/police.module.ts [hunk 21]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/stats/modules/police/police.module.ts:2]: add row

File: iTechArt.UI/src/app/stats/modules/students/students.module.ts [hunk 22]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/app/stats/modules/students/students.module.ts:2]: add row

File: iTechArt.UI/src/index.html [hunk 23]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Government</title>
```
Comment (andrei-shpileuski) [inline iTechArt.UI/src/index.html]: change to government's statistics. And add some description. by simple 5-10 words about app (self-education)

File: iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts (hunk 16 above)
Comment (AlbinaPavlova) [reply iTechArt.UI/src/app/shared/pages/dashboard-page/dashboard-page.component.ts]: why?

File: iTechArt.UI/src/index.html (hunk 23 above)
Comment (AlbinaPavlova) [reply iTechArt.UI/src/index.html]: should i add description in title, if it's already on site under title?

**Pull request merged**